### DISCUSSION TITLE
[Question]: how do you refine max_area or min_area filters in 0.14?

### DISCUSSION THREAD

**Discussion by davison** [blakeblackshear/frigate#13541] [Ask A Question]
### What is your question?

In 0.13 you could use the snapshots to see the area of a motion box or bounding box and use this to set values for filters like `max_area` or `min_area`. How do you now do that in 0.14 - where can I see the area of a box that triggered a detection so that I can refine these values?

**Comment by NickM-27** [comment]
This is covered in the FAQ, there is no screen for this. The screen has already been added for 0.15
